### ISSUE TITLE
Rows per page displays incorrectly in IE 11 and Microsoft Edge on page load

### ISSUE THREAD

**Issue by khe817** [xaksis/vue-good-table#228]
### Issue Type

- [x] Bug

### Specs
What version are you using?
2.0.1

What browser?
IE 11 and Microsoft Edge

### Expected Behavior
Using this setting: `:pagination-options="{ enabled: true, perPage: 10, }"` , expect "Rows per page" display correctly on page load, which is `10`.

### Actual Behavior
When the page loads initially, in pagination, "Rows per page" is set to `All` instead of `10` like it should. However, clicking on Next, or select a different "Rows per page" will set it back to being functional normally.

### Steps to Reproduce the Problem
Use this config on any sample data that has more than 10 rows:

```html
<div id="app">
  <vue-good-table
    :columns="columns"
    :rows="rows"
    :pagination-options="{ enabled: true, perPage: 10, }"
    > 
  </vue-good-table>                  
</div>
```

### Additional info:
If I set `dropdownAllowAll: false,` in `:pagination-options` then "Rows per page" is set to `10` correctly on page load.

### jsfiddle
Can't. jsfiddle.com webpage does not load properly for neither IE 11 or Edge. Lol.

**Comment by xaksis** [comment]
> Can't. jsfiddle.com webpage does not load properly for neither IE 11 or Edge. Lol.
hahaha... jeez @khe817 i'll try to reproduce it on my end... somehow.

**Comment by xaksis** [comment]
https://twitter.com/michlbrmly/status/981855020948877312

Even github is dropping support for IE

**Comment by xaksis** [comment]
that took a few attempts. but this is now fixed in v2.0.5

**Comment by khe817** [comment]
Bug is confirmed fixed.

Thank you very much!
